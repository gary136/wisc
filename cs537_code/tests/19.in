loop 2 echo $loop hello world
path /bin /usr/bin/
loop 3 head -n $loop tests/19.in
