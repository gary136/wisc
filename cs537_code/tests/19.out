1 hello world
2 hello world
loop 2 echo $loop hello world
loop 2 echo $loop hello world
path /bin /usr/bin/
loop 2 echo $loop hello world
path /bin /usr/bin/
loop 3 head -n $loop tests/19.in
